templates/base.html

```diff
@@ -53,6 +53,9 @@
             <li class="nav-item">
               <a class="nav-link" href="/Mine">Mine</a>
```

Comment:
What Mine will route too? Please think about the routing to login and from login to "mine"

Reply:
decided to route the button Mine to the login/ url to keep it simple, and after the user will be log in with his House ID, the page will direct him to his Mine view page